nav2_costmap_2d/src/collision_checker.cpp

```diff
@@ -34,12 +34,14 @@ CollisionChecker::CollisionChecker(
   FootprintSubscriber & footprint_sub,
   tf2_ros::Buffer & tf,
   std::string name,
-  std::string global_frame)
+  std::string global_frame,
```

Comment:
```suggestion
  std::string & global_frame,
```

Reply:
There is a default value for the `global_frame` argument in the declaration of `CollisionChecker::CollisionChecker` in [collision_check.hpp](https://github.com/Marwan99/navigation2/blob/master/nav2_costmap_2d/include/nav2_costmap_2d/collision_checker.hpp#L42). Do want to get rid of the default value?

Reply:
ah good point.